getArgv.js

```diff
@@ -0,0 +1,179 @@
+function getArgv() {
+    const ogsPvAIs = ["LeelaZero", "Sai", "KataGo", "PhoenixGo", "Leela"];
```

Comment:
We do not want to duplicate this. We could make a const.js files or something like that that exports common constants like this.

Reply:
yes, duplication is bad

it could also be useful for other parts of the code where we reuse and redeclare shared constants